x/auth/client/cli/broadcast.go

```diff
@@ -26,6 +27,11 @@ $ <appcli> tx broadcast ./mytxn.json
 		Args: cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
```

Comment:
This should be turned into a standalone function to make it easier to test the functionality and achieve comprehensive code coverage. @jgimeno pleas take inspiration on how test cmds with mocked std{err,in,out} from the commands in the `client/keys` package.